### PR TITLE
[17.0][MIG] account_payment_return_import_iso20022: Migration to 17.0 

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- Standard migration to v17

@Tecnativa
TT49284

@pedrobaeza @carlos-lopez-tecnativa

### CODE REVIEW

File: account_payment_return_import_iso20022/wizard/camt_parser.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def add_value_from_node(ns, node, xpath_str, obj, key, join_str=None):
         If xpath_str is a list (or iterable), it will be seen as a series
         of search path's in order of preference. The first item that results
         in a found node will be used to set a value in the dictionary."""
-        if not isinstance(xpath_str, (list, tuple)):
+        if not isinstance(xpath_str, list | tuple):
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/wizard/camt_parser.py:43]: Why this change? It was working before and AFAIK, this hasn't changed across Python versions.

Comment (carolinafernandez-tecnativa) [reply account_payment_return_import_iso20022/wizard/camt_parser.py:43]: If not changed, pre-commit fails.

Check error message:

account_payment_return_import_iso20022/wizard/camt_parser.py:43:16: UP038 Use `X | Y` in `isinstance` call instead of `(X, Y)`

Comment (pedrobaeza) [reply account_payment_return_import_iso20022/wizard/camt_parser.py:43]: Oh, I didn't know. It seems then a legacy notation that is still compatible...

**Review by pedrobaeza — APPROVED** [review]

**Review by carlos-lopez-tecnativa — APPROVED** [review]
LGTM

**Comment by carlos-lopez-tecnativa** [comment]
@carolinafernandez-tecnativa Please update the latest commit; the dependency has now been merged. https://github.com/OCA/account-payment/pull/752

**Comment by carolinafernandez-tecnativa** [comment]
> @carolinafernandez-tecnativa Please update the latest commit; the dependency has now been merged. #752

done~

**Comment by carlos-lopez-tecnativa** [comment]
> > @carolinafernandez-tecnativa Please update the latest commit; the dependency has now been merged. #752
> 
> done~

@carolinafernandez-tecnativa, sorry for the confusion. I initially thought this module depended on `account_payment_return`, but it actually depends on [account_payment_return_import](https://github.com/OCA/account-payment/pull/753), which has not been merged yet. Please update the PR description to add the dependency and avoid confusion. Also, add the dependency again. Sorry for the mix-up.

image
